mv/plan/src/main/scala/org/apache/carbondata/mv/plans/modular/Modularizer.scala

```diff
@@ -66,6 +68,22 @@ abstract class Modularizer[TreeType <: TreeNode[TreeType]] {
     makeupAliasMappings(mplans)
   }
 
+  private def transformPlan(plan: LogicalPlan) = {
```

Comment:
please change the method name to something meaningful according to functionality